internal/pkg/config/input.go

```diff
@@ -57,6 +57,7 @@ type Server struct {
 	MaxHeaderByteSize int               `config:"max_header_byte_size"`
 	RateLimitBurst    int               `config:"rate_limit_burst"`
 	RateLimitInterval time.Duration     `config:"rate_limit_interval"`
+	HardConnLimit     int               `config:"hard_conn_limit"`
```

Comment:
Is the `hard_` prefix needed.

What about `max_connections`?

Reply:
sure.